lib/bank.js

```diff
@@ -20,7 +20,13 @@ function bank(system) {
 				type: 'textinput',
 				id: 'text',
 				label: 'Button text',
-				width: 12
+				width: 10
+			},
+			{
+				type: 'alignmentcontrol',
```

Comment:
I think the better solution would be to have a type: 'select' or 'dropdown'. And then give the options as options: [ { id: xx, value: 'Left' }, ... ]